packages/hint-compat-api/src/css.ts

```diff
@@ -223,6 +237,37 @@ const walk = (ast: ContainerBase, context: Context) => {
     reportUnsupported(reportsMap, context);
 };
 
+export const formatAlternates = (language: string, unsupported: UnsupportedBrowsers, formatFeature?: (name: string) => string): string[] => {
+    const groupedAlternates = new Map<string, BrowserAlternateDetails[]>();
+
+    for (const browser of unsupported.browsers) {
+        const details = unsupported.details.get(browser);
+
+        if (details && details.alternate) {
+            const { name } = details.alternate;
+            const group = groupedAlternates.get(name) || [];
+
+            group.push({ browser, ...details.alternate });
+            groupedAlternates.set(name, group);
+        }
+    }
+
+    const alternateMessages = new Set<string>();
+
+    for (const [name, list] of groupedAlternates) {
+        const formattedName = formatFeature ? formatFeature(name) : name;
+        const browsers = list.map(({ browser, versionAdded, versionRemoved }) => {
```

Comment:
```suggestion
        const browsers = list.map(formatSupported);
```

Reply:
That won't work unless I also update the signature for `formatSupported` to take an object (which I'd prefer not to do here).